js/ui/calendar/ui.calendar.base_view.js

```diff
@@ -152,6 +160,26 @@ const BaseView = Widget.inherit({
         params.cellDate = this._getNextCellData(cellDate);
     },
 
+    _renderWeekNumberCell: function(rowData) {
```

Comment:
How [cellTemplate](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#cellTemplate) option should work for this cells? What parameters it accepts? `data: undefined, weekNumber: number`? 

We should write tests for this and update documentation.

Reply:
should it work for this cells?

Reply:
I think we should have a capability to customize these cells

Reply:
implemented

Reply:
throw to the trash can and started again

Reply:
nope seem good, implemented and added a quinit + one screenshot test for templates

Reply:
Let's add the commit hash please to see the changes where is a implementation

Reply:
no hash, parts in two pr (3 and 4), implemented here: https://github.com/DevExpress/DevExtreme/pull/22674/files#diff-5efeafa614b53cfce71ef52ae5b44108e26958f71e4efb4d908a0bd81689f029R176 and herehttps://github.com/DevExpress/DevExtreme/pull/22674/files#diff-5efeafa614b53cfce71ef52ae5b44108e26958f71e4efb4d908a0bd81689f029R211